src/components/donation/Steps/Splits/Splits.tsx

```diff
@@ -50,16 +50,16 @@ export default function Split({
 
       {/** percentages */}
       <div className="flex justify-between mt-6 mb-3">
-        <div className="grid gap-1 grid-cols-[auto_1fr] items-center">
-          <Image src={leaf} className="mr-1" />
-          <p className="text-xl">{lockedSplitPct}%</p>
-          <p className="uppercase text-xs col-span-full">Sustainable fund</p>
-        </div>
         <div className="grid gap-1 grid-cols-[1fr_auto] items-center">
           <p className="text-xl text-right">{liqSplitPct}%</p>
           <Image src={sendMoney} className="ml-1" />
           <p className="uppercase text-xs col-span-full">Instantly Available</p>
         </div>
+        <div className="grid gap-1 grid-cols-[auto_1fr] items-center">
+          <Image src={leaf} className="mr-1" />
+          <p className="text-xl">{lockedSplitPct}%</p>
+          <p className="uppercase text-xs col-span-full">Sustainable fund</p>
+        </div>
       </div>
 
       {/** slider */}
```

Comment:
There's a bug - the slider is increasing, but the displayed liquid value is decreasing:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/456e524f-6458-4d28-806f-a9dd808bb8fd)

The problem is that the slider is still tracking the locked value instead of the liquid one.
- update local state to `const [liquidSplitPct, setLiquidSplitPct] = useState(persistedLiqSplit);`
- remove `lockedSplitPct` (to display the value above the slider simply use `100 - liquidSplitPct`)
- update `Slider.Root` to:
```ts
      <Slider.Root
        value={[liquidSplitPct]}
        onValueChange={([pct]) => setLiquidSplitPct(pct)}
        className="relative flex items-center select-none touch-none my-2"
      >
```

Reply:
@misicnenad When the slider moves to the right, the user is increasing the locked % amount desired, so it follows that there _should_ be a complementary decrease the liquid % and liquid amount values. I don't think there's a need to adjust that logic. We've only switched the display order, so that the slider color matches the color of the icon it ends up underneath at the end of either range. Lmk if that tracks, thanks!

Reply:
@SovereignAndrey I think donators would find this confusing - the left-hand side of the slider should represent whatever % is displayed _on the left_. As a donator I'd expect the left % match the size of the green (left) part of the slider.

IMO the current setup is the same as if the user hit "Page Up" key on their keyboard, but their screen scrolled *down*.